corehq/apps/cleanup/tasks.py

```diff
@@ -23,14 +23,17 @@
 logger = logging.getLogger(__name__)
 
 
-@periodic_task(run_every=crontab(minute=0, hour=0), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
```

Comment:
I don't think this is essential (hence why I approved it) but I wonder if we should just comment this out and add a comment that this is temporarily disabled/work is pending. I doubt we will run into an issue with someone unaware of the pending work attempting to cleanup a seemingly unused function,

Reply:
That's why I added the comment in the body which I'm now realizing is a little vague but I can do that and make it clearer that this task will be restructured at a later date.

Reply:
Thanks for the update!